x-pack/apm-server/sampling/eventstorage/storage.go

```diff
@@ -126,19 +130,10 @@ func (rw *ReadWriter) Close() {
 // files exceeds the configured threshold.
 func (rw *ReadWriter) Flush(limit int64) error {
```

Comment:
We should remove the limit param, right? Also the doc comment needs updating since ErrLimitReached won't be returned any more.